CHANGELOG.rst

```diff
@@ -1,13 +1,24 @@
-1.0.1 (Unreleased)
+1.1 (Unreleased)
 ==================
```

Comment:
underline is now too long.

Reply:
Pesky underlining!!  Should now be fixed (with a sneaky force push ;) )